src/main/java/be/seeseemelk/mockbukkit/entity/ThrowableProjectileMock.java

```diff
@@ -0,0 +1,24 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.ServerMock;
+import org.bukkit.entity.ThrowableProjectile;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+public abstract class ThrowableProjectileMock extends ProjectileMock implements ThrowableProjectile
+{
+
+	protected ThrowableProjectileMock(@NotNull ServerMock server, @NotNull UUID uuid)
+	{
+		super(server, uuid);
+	}
+
+	@Override
+	public abstract @NotNull ItemStack getItem();
+
+	@Override
+	public abstract void setItem(@NotNull ItemStack item);
```

Comment:
Are these necessary? If removed inheriting classes will still have to override them.

And if the implementation is the same for everything that inherits, can't they be implemented here?